images/dind/docker.service

```diff
@@ -1,15 +0,0 @@
-[Unit]
-Description=Docker-in-Docker (dind)
-After=network.target
-
-[Service]
-Type=notify
-EnvironmentFile=-/etc/sysconfig/docker-network
```

Comment:
dind.conf is additive to the default docker.service, and that line does appear in it.  The only way that line would be removed would be if I set 'EnvironmentFile=' in dind.conf to reset that collection.